.github/workflows/int-and-connected-test-run.yml

```diff
@@ -1,103 +1,42 @@
-    branches:
-      - develop
-    paths:
-      - '!*'
-      - 'src/**'
-      - 'gradle**'
-      - 'gradle/**'
-      - '**.gradle'
-      - 'Dockerfile'
-      - 'datarepo-clienttests/**'
-      - '.github/workflows/int-and-connected-test-run.yml'
-      - '.swagger-codegen-ignore'
   workflow_dispatch: {}
+  pull_request:
   schedule:
     - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
+# do not allow concurrent runs of this workflow on the same branch
 concurrency:
-  # Don't run this workflow concurrently on the same branch
   group: ${{ github.workflow }}-${{ github.ref }}
-  # For PRs, don't wait for completion of existing runs, cancel them instead
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 jobs:
-  test_check:
-    name: "Checkout, verify and run unit tests"
-    outputs:
-      job-status: ${{ job.status }}
+  test_unit:
+    name: Run unit tests
+    runs-on: ubuntu-latest
     timeout-minutes: 60
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    if: "!contains( github.event.pull_request.labels.*.name, 'skip-ci')"
-    runs-on: ${{ matrix.os }}
-    ## skips if pr label is 'skip-ci'
-    # run a local Postgres container in Docker for the basic check tests
-    services:
-      postgres:
-        image: postgres:11
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: postgres
-        ports:
-          - 5432:5432
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
     steps:
-      - name: "Checkout code"
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: "Cache build"
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.CACHE_PATHS }}
-          key: ${{ runner.os }}-build-unit }}
-      - name: "Run unit tests and sonar scan via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        with:
-          actions_subcommand: 'gradleinttest'
-          pgport: ${{ job.services.postgres.ports[5432] }}
-          test_to_run: 'check'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-          sonar_token: ${{ secrets.SONAR_TOKEN }}
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'gradle'
+      - name: Run unit tests
+        env:
+          GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          TDR_LOG_APPENDER: 'Console-Standard'
+        run: |
+          # extract service account credentials
+          base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
+          # assemble code, run unit tests, and generate scan
+          ./gradlew assemble
+          ./gradlew check --scan jacocoTestReport sonar
```

Comment:
Here is the minimal amount of code required to run unit tests in a hermetic environment. No more `datarepo-actions`!

Reply:
I did this to mimic the current behavior: https://github.com/broadinstitute/datarepo-actions/blob/master/actions/main/src/gradleinttest.sh#L61

Reply:
I like @pshapiro4broad 's suggestions here, and agree with making any behavior changes here in a future PR.

Reply:
I'll roll in these changes in the integration PR since the feedback cycle running connected tests and integration tests is very slow.